src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,102 @@
+const {
+  filterExpenses,
+  addExpense,
+  getExpense,
+  deleteExpense,
+  updateExpense,
+  getExpenses,
+} = require('../services/expenses.servise');
+const { getUser } = require('../services/users.service');
+
+const getAll = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  if (userId || categories || from || to) {
+    res.statusCode = 200;
+    res.send(filterExpenses(req.query));
+
+    return;
+  }
+
+  const expenses = getExpenses();
```

Comment:
The `getExpenses` function is called without any arguments. Ensure that this aligns with the expected parameters of the `getExpenses` function in the service layer.